strongbox-user-management/src/main/java/org/carlspring/strongbox/authentication/registry/support/AuthenticatorsRegistrySerializer.java

```diff
@@ -0,0 +1,43 @@
+package org.carlspring.strongbox.authentication.registry.support;
+
+import org.carlspring.strongbox.authentication.api.Authenticator;
+import org.carlspring.strongbox.authentication.registry.AuthenticatorsRegistry;
+
+import java.io.IOException;
+import java.util.List;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import com.fasterxml.jackson.databind.ser.std.StdSerializer;
+
+/**
+ * @author Przemyslaw Fusik
+ */
+public class AuthenticatorsRegistrySerializer
```

Comment:
Yes, I noticed that. But if I use JAXB then I need to use parser explicitly and deserialize it explicitly .... so much work to do :P By using JAXB it does not serialize/deserialize automatically (at least in our current implementation). 

Above solution (using @JsonSerialize from jackson) allows us to skip the noise (explicit parsing and serializing) and everything works under the hood automatically.

If you want it - I will change it as you requested.